resources/windows/wix/service.component.wxs.erb

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+  <%- get_services.each do |service| -%>
+    <%- service.options.each do |component| -%>
+    <DirectoryRef Id="<%= component[:directory_ref] %>">
+      <Component Id='<%= component[:id] %>' Guid="<%= component[:guid] %>" <%= "Win64=\"$(var.Win64)\"" unless component[:win64].nil? %>>
+        <%- if component[:file][:id] == "NSSM" -%>
```

Comment:
@melissa As an approach, this looks very good.
I still need to try it in practice against build so that I can ascertain the the syntax is correct.
(unfortunately there was rather too much disruption today to do it).
Will do this first thing on Monday.

In the meantime would value comments from @Iristyle @ferventcoder and @MikaelSmith